packages/ember-intl/index.js

```diff
@@ -57,6 +57,32 @@ module.exports = {
     return this.app;
   },
 
+  outputPaths: function() {
+    var assetPath = 'assets/intl';
```

Comment:
`assets` should come from some whatever the the path that treeForPublic is written to, I believe there is an option where this can be configurable so we don't want to make assets static.